diff --git a/node_modules/@wordpress/block-library/build-module/image/image.mjs b/node_modules/@wordpress/block-library/build-module/image/image.mjs
index 4b78108..bbc4160 100644
--- a/node_modules/@wordpress/block-library/build-module/image/image.mjs
+++ b/node_modules/@wordpress/block-library/build-module/image/image.mjs
@@ -332,7 +332,7 @@ function Image({
   const hasNonContentControls = blockEditingMode === "default";
   const isContentOnlyMode = blockEditingMode === "contentOnly";
   const showDimensionsControls = allowResize && hasNonContentControls;
-  const isResizable = allowResize && hasNonContentControls && !isWideAligned && isLargeViewport;
+  const isResizable = allowResize && hasNonContentControls && !isWideAligned;
   const imageSizeOptions = imageSizes.filter(
     ({ slug }) => image?.media_details?.sizes?.[slug]?.source_url
   ).map(({ name, slug }) => ({ value: slug, label: name }));
