diff --git a/node_modules/@wordpress/block-library/build-module/image/image.js b/node_modules/@wordpress/block-library/build-module/image/image.js
index 14d14ab..6bab7fc 100644
--- a/node_modules/@wordpress/block-library/build-module/image/image.js
+++ b/node_modules/@wordpress/block-library/build-module/image/image.js
@@ -311,7 +311,7 @@ function Image({
   const [hasImageErrored, setHasImageErrored] = useState(false);
   const hasNonContentControls = blockEditingMode === "default";
   const isContentOnlyMode = blockEditingMode === "contentOnly";
-  const isResizable = allowResize && hasNonContentControls && !isWideAligned && isLargeViewport;
+  const isResizable = allowResize && hasNonContentControls && !isWideAligned;
   const imageSizeOptions = imageSizes.filter(
     ({ slug }) => image?.media_details?.sizes?.[slug]?.source_url
   ).map(({ name, slug }) => ({ value: slug, label: name }));
