diff --git a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
index aa2e9a5..f2d8481 100644
--- a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
+++ b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
@@ -192,6 +192,7 @@ class Inserter extends Component {
         shift: true
       },
       onToggle: this.onToggle,
+      open: this.props.open,
       expandOnMobile: true,
       headerTitle: __('Add a block'),
       renderToggle: this.renderToggle,
