diff --git a/node_modules/@wordpress/block-editor/build-module/components/inserter-list-item/index.js b/node_modules/@wordpress/block-editor/build-module/components/inserter-list-item/index.js
index 89a3bbc..8564798 100644
--- a/node_modules/@wordpress/block-editor/build-module/components/inserter-list-item/index.js
+++ b/node_modules/@wordpress/block-editor/build-module/components/inserter-list-item/index.js
@@ -28,6 +28,10 @@ function InserterListItem({
   ...props
 }) {
   const isDraggingRef = useRef(false);
+  const isTouchDeviceRef = useRef(
+    window.matchMedia( '(hover: none)' ).matches ||
+    window.matchMedia( '(pointer: coarse)' ).matches
+  );
   const itemIconStyle = item.icon ? {
     backgroundColor: item.icon.background,
     color: item.icon.foreground
@@ -80,7 +84,7 @@ function InserterListItem({
           }
         },
         onMouseEnter: () => {
-          if (isDraggingRef.current) {
+          if (isDraggingRef.current || isTouchDeviceRef.current) {
             return;
           }
           onHover(item);
diff --git a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
index 78b9b8f..9a8efb8 100644
--- a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
+++ b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
@@ -187,10 +187,12 @@ class Inserter extends Component {
         'is-quick': isQuick
       }),
       popoverProps: {
+        ...this.props.popoverProps,
         position,
         shift: true
       },
       onToggle: this.onToggle,
+      open: this.props.open,
       expandOnMobile: true,
       headerTitle: __('Add a block'),
       renderToggle: this.renderToggle,
diff --git a/node_modules/@wordpress/block-editor/build-module/components/provider/index.js b/node_modules/@wordpress/block-editor/build-module/components/provider/index.js
index e89fa49..0f14ae8 100644
--- a/node_modules/@wordpress/block-editor/build-module/components/provider/index.js
+++ b/node_modules/@wordpress/block-editor/build-module/components/provider/index.js
@@ -104,7 +104,7 @@ export const ExperimentalBlockEditorProvider = withRegistryProvider(props => {
 export const BlockEditorProvider = props => {
   return /*#__PURE__*/_jsx(ExperimentalBlockEditorProvider, {
     ...props,
-    stripExperimentalSettings: true,
+    stripExperimentalSettings: false,
     children: props.children
   });
 };
