diff --git a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
index 78b9b8f..3c0049d 100644
--- a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
+++ b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
@@ -191,6 +191,7 @@ class Inserter extends Component {
         shift: true
       },
       onToggle: this.onToggle,
+      open: this.props.open,
       expandOnMobile: true,
       headerTitle: __('Add a block'),
       renderToggle: this.renderToggle,
diff --git a/node_modules/@wordpress/block-editor/build-module/components/provider/index.js b/node_modules/@wordpress/block-editor/build-module/components/provider/index.js
index e89fa49..0f14ae8 100644
--- a/node_modules/@wordpress/block-editor/build-module/components/provider/index.js
+++ b/node_modules/@wordpress/block-editor/build-module/components/provider/index.js
@@ -104,7 +104,7 @@ export const ExperimentalBlockEditorProvider = withRegistryProvider(props => {
 export const BlockEditorProvider = props => {
   return /*#__PURE__*/_jsx(ExperimentalBlockEditorProvider, {
     ...props,
-    stripExperimentalSettings: true,
+    stripExperimentalSettings: false,
     children: props.children
   });
 };
