diff --git a/node_modules/@wordpress/editor/build-module/private-apis.js b/node_modules/@wordpress/editor/build-module/private-apis.js
index eb169ff..2b8528b 100644
--- a/node_modules/@wordpress/editor/build-module/private-apis.js
+++ b/node_modules/@wordpress/editor/build-module/private-apis.js
@@ -9,6 +9,7 @@ import * as interfaceApis from '@wordpress/interface';
 import { lock } from './lock-unlock';
 import { EntitiesSavedStatesExtensible } from './components/entities-saved-states';
 import EditorContentSlotFill from './components/editor-interface/content-slot-fill';
+import useBlockEditorSettings from './components/provider/use-block-editor-settings';
 import BackButton from './components/header/back-button';
 import CreateTemplatePartModal from './components/create-template-part-modal';
 import Editor from './components/editor';
@@ -43,6 +44,7 @@ lock(privateApis, {
   ResizableEditor,
   registerCoreBlockBindingsSources,
   // This is a temporary private API while we're updating the site editor to use EditorProvider.
+  useBlockEditorSettings,
   interfaceStore,
   ...remainingInterfaceApis
 });
