diff --git a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
index aa2e9a5..a38d989 100644
--- a/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
+++ b/node_modules/@wordpress/block-editor/build-module/components/inserter/index.js
@@ -188,10 +188,12 @@ class Inserter extends Component {
         'is-quick': isQuick
       }),
       popoverProps: {
+        ...this.props.popoverProps,
         position,
         shift: true
       },
       onToggle: this.onToggle,
+      open: this.props.open,
       expandOnMobile: true,
       headerTitle: __('Add a block'),
       renderToggle: this.renderToggle,
